README.md

```diff
@@ -13,7 +13,8 @@ By the end of this, students should be able to:
 
 -   Give two reasons why inheritance and Mixins are desirable.
 -   Write a class which inherits from another class.
--   Describe why mixins are preferred to inheritance.
+-   Write a mixin
+-   Describe the difference between inheritance, composition, and mixins
```

Comment:
I think your use of an oxford comma here is correct, but the talks header should match this. I opened an issue, so no need to address it in this PR.